gemsfx-demo/src/main/java/com/dlsc/gemsfx/demo/CalendarPickerApp.java

```diff
@@ -26,7 +26,7 @@ public void start(Stage stage) {
         calendarPicker.setValue(LocalDate.now());
 
         Label valueLabel = new Label();
-        valueLabel.textProperty().bind(Bindings.createStringBinding(() -> calendarPicker.getConverter().toString(calendarPicker.getValue()), calendarPicker.valueProperty()));
+        valueLabel.textProperty().bind(Bindings.createStringBinding(() -> calendarPicker.getValue() == null ? "No Date Selected" : calendarPicker.getConverter().toString(calendarPicker.getValue()), calendarPicker.valueProperty()));
```

Comment:
I am not a fan of having messages like that because it requires internationalisation.